gradio/components.py

```diff
@@ -74,6 +73,10 @@
 from gradio.interpretation import NeighborInterpretable, TokenInterpretable
 from gradio.layouts import Column, Form, Row
 
+if not wasm_utils.is_wasm:
+    # TODO: Support ffmpeg on Wasm
+    from ffmpy import FFmpeg
```

Comment:
Probably would be good to raise an error in `gr.Video()` where `FFmpeg` is used  explaining that certain features are not available in wasm.

Reply:
I added `wasm_utils.WasmUnsupportedError` and made `gr.Video` to raise it when ffmpeg is to be used 👍